app/globals.css

```
@@ -161,11 +161,36 @@ body {
+    black 10%,
+    black 90%,
+    transparent 100%
+  );
+
+  -webkit-mask-image: linear-gradient(
+    to bottom,
+    transparent 0%,
+    black 10%,
+    black 90%,
+    transparent 100%
+  );
```

Stylelint (17.5.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)